state/state.go

```diff
@@ -219,8 +232,10 @@ func (s *State) Bytes() []byte {
 	return wire.BinaryBytes(s)
 }
 
-// SetBlockAndValidators mutates State variables to update block and validators after running EndBlock.
-func (s *State) SetBlockAndValidators(header *types.Header, blockPartsHeader types.PartSetHeader, abciResponses *ABCIResponses) {
+// SetBlockAndValidators mutates State variables to update block and validators after running
+// EndBlock.
+func (s *State) SetBlockAndValidators(header *types.Header, blockPartsHeader types.PartSetHeader,
```

Comment:
are these changes all being made automatically? what's the setting? we should make a firm decision on this so we don't have endless PRs making small formatting changes

Reply:
I'm following the go convention of lines less than 100. The gometalinter has the linter `lll` which picks this up. I don't think we have to change it all at once, but for all new code we should aim to have a width of less than 100.

It follows the go convention and for a good reason. Working on split screen is much nicer with line breaks at 100.

@zramsay Could we add this to the coding repo?